09 Palindromes: Non Regex Solution Added

Hifilo opened the pull request:
<!-- Thank you for taking the time to contribute to The Odin Project. In order to get this pull request (PR) merged in a reasonable amount of time, you must complete this entire template. -->

## Because
<!-- Summarize the purpose or reasons for this PR, e.g. what problem it solves or what benefit it provides. -->
Since the readme didn't suggest regex, I got the impression we were to do this wihtout regex. But after finishing, and checking the solution, there was only a regex version.

## This PR
<!-- A bullet point list of one or more items describing the specific changes. -->

- Regex Method commented
-Non Regex method with comments attached.

## Issue
<!--
If this PR closes an open issue in this repo, replace the XXXXX below with the issue number, e.g. Closes #2013.

If this PR closes an open issue in another TOP repo, replace the #XXXXX with the URL of the issue, e.g. Closes https://github.com/TheOdinProject/curriculum/issues/XXXXX

If this PR does not close, but is related to another issue or PR, you can link it as above without the 'Closes' keyword, e.g. 'Related to #2013'.
-->
Closes #XXXXX

## Additional Information
<!-- Any other information about this PR, such as a link to a Discord discussion. -->

## Pull Request Requirements
<!-- Replace the whitespace between the square brackets with an 'x', e.g. [x]. After you create the PR, they will become checkboxes that you can click on. -->
-   [x ] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `01_helloWorld: Update test cases`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If this PR includes changes that needs to be updated on the `solutions` branch, I have created another PR (and linked it to this PR).

rlmoser99 reviewed (changes requested):
Thank you for taking the time to make this update. Since we do not teach regex in Foundations or truly recommend it in our curriculum, I believe your solution should replace the current solution.

I know it has been quite some time since you submitted this PR, but would you be willing to make some changes? In addition to removing the original solution, could you remove the comments. I believe your solution is very readable to understand.

Hifilo commented:
> Thank you for taking the time to make this update. Since we do not teach regex in Foundations or truly recommend it in our curriculum, I believe your solution should replace the current solution.
> 
> I know it has been quite some time since you submitted this PR, but would you be willing to make some changes? In addition to removing the original solution, could you remove the comments. I believe your solution is very readable to understand.

Hi, no worries. Yes it has been a minute, and I forgot about this. 

To be fair, I wanted to use a regex method. but because it wasn't instructed to do so, I thought I was supposed to find the solution without it. 
So when I checked the solutions after I passed all tests, I was hoping to see a cleaner method, than what I had originally written. I saw the regex method which kind of left me a little perplexed. Thats really the only reason I added the solution.
Another user from another discord suggested putting the alphabet in a variable, and using filter, and that led to this solution. 

I've never gone through this review process before, so I'm not sure if I did it correctly, but I updated the code with the changes you requested, and sent another commit.

rlmoser99 reviewed (approved):
This looks great! Thanks for taking the time to come back to this PR and make these change!
